### ISSUE TITLE
Signed distance - How to increase performance ?

### ISSUE THREAD

**Issue by MagicFeeling** [mikedh/trimesh#1715]
Hello,
thank you for this amazing library. Its really amazing. Currently I am using the signed distance function a lot and I was wondering if there is a way of how to increase performance. I need to test chunks of between 1000 and 2000 coordinates which take quite some time. Is there any hardware suggestion I could get in order to increase the speed? (Faster RAM or faster CPU) or is there any GPU support ?
Thank you in advance

**Comment by mikedh** [comment]
Hey, glad it's useful! Contains tests are probably using trimesh's tragically slow but accurate numpy based ray check, one of the wishlist items in #1557 is speed-ups for the native ray checks. 

You could also install embree (which does have some precision issues like #242) but is ~50x faster. If you're on linux you can probably install it with [this bash script](https://github.com/mikedh/trimesh/blob/main/docker/builds/embree.bash).

Speedup PR's for the native raytracer also super welcome! Check out [the contributing guide](https://trimsh.org/contributing.html), running your script under pyinstrument and embedding at hotspots is probably a good way to start looking into it.

**Comment by MagicFeeling** [comment]
Thank you for your response. I tried to use embree but it seems not working for me. I am using a conda environment and installed it with [conda - embree](https://anaconda.org/conda-forge/embree). Is embree processor dependent or is there anything else I need to install. I also installed trimesh with Anaconda. The statement `if ray.has_embree and use_embree:` in the file base.py seems to fail. I set use_embree to true.
